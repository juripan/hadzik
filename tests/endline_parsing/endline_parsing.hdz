cif x = 8
{cif x = 8}{
    cif d = 12
}{
    cif x = 7}
{{}}
kim 1 {
    kim(1 == 1){
        konec
    }
    konec
    x++
} 
vychod(x)
