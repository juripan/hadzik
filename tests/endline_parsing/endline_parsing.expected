stdout: | stderr: | returncode: 8